TimberAPI/EntityLinkerSystem/EntityLinker.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Timberborn.ConstructibleSystem;
+using Timberborn.Persistence;
+using TimberbornAPI.Internal;
+using UnityEngine;
+
+namespace TimberbornAPI.EntityLinkerSystem
+{
+    /// <summary>
+    /// Defines the behaviour of Linkers
```

Comment:
Can we add a description of how you can use this here?

Reply:
Yes